packages/jellyfish-api-core/src/category/account.ts

```diff
@@ -235,6 +235,23 @@ export class Account {
   async utxosToAccount (payload: UtxosToAccountPayload, utxos: UtxosToAccountUTXO[] = []): Promise<string> {
     return await this.client.call('utxostoaccount', [payload, utxos], 'number')
   }
+
+  /**
+   * Returns count of account history
+   *
+   * @param {OwnerType} [owner='mine'] single account ID (CScript or address) or reserved words 'mine' to list history count for all owned accounts or 'all' to list whole DB
+   * @param {AccountHistoryCountOptions} [options]
+   * @param {boolean} [options.no_rewards]  Filter out rewards
+   * @param {string} [options.token]  Filter by token
+   * @param {string} [options.txtype]  Filter by transaction type, supported letter from 'CRTMNnpuslrUbBG'
+   * @return {Promise<number>}  Count of account history
+   */
+  async accountHistoryCount (
+    owner: OwnerType = 'mine',
```

Comment:
I think we can replace `OwnerType` with an enum. Currently it's only 'mine' or 'all' but OwnerType is accepting 'mine', 'all' or any string. Maybe we can restrict it to an enum for better strict checking.

Reply:
I saw there was an owner type already in place that i why i used that. Let me change it to an enum as you suggested.

Reply:
@thedoublejay, from ain/rcp_account.cpp, `accountHistory` ownerType can accept the reserved words , `mine` and `all ` or a string  `Single account ID (CScript or address) or reserved words`. So using an enum is not ideal. What do you think?

Reply:
@siradji hmm yeah good point. In this case we can wrap it in an interface. Something like OwnerType | string. Then at least on other parts of code, we can use OwnerType.MINE.

Reply:
Good point. Lemme do that.

Reply:
@canonbrother, changing `Ownertype` to an `enum`  will after `listAccount` implementation that you worked on. How should i resolve this?

Reply:
yea its good to go.. refactor it  =)